project_template/config/nightwatch.js

```diff
@@ -30,7 +30,7 @@ module.exports = {
       desiredCapabilities: {
         browserName: "chrome",
         chromeOptions: {
-          args: ["--no-sandbox"]
+          args: ["--no-sandbox", "--headless"]
```

Comment:
So these capabilities are only enabled when CI=true is set, right? I wonder if it would be useful to run headless by default locally too, and perhaps allow to override via some env var. WDYT? /cc @leonderijke

Reply:
I think it would indeed be useful to default to headless.